log/logfields/fields.go

```diff
@@ -0,0 +1,48 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package logfields
```

Comment:
`logfields` seems duplicated (package `log/logfields` v.s. `log/fields`) and a little verbose when used (like `logfields.PodSandboxID`), and people need to import 2 packages if they want to log something (`log` and `log/logfields`).

How about:

1. Just put them as const in the `log` package itself (`log/fields.go`), so people only import `log` package and use, e.g., `log.Snapshotter`. Or,
2. Rename package `log/logfields` to `log/fields`?

Reply:
I trialed 1, I didn't mind it too much but felt fields helped to distinguish the log packages current job which is small and useful wrappers around our current structured logging solution. I can revert back to that

Reply:
that might be just my preference, we can see others' suggestions before revert back :)

Reply:
Lets go the no new package route, and I'll list the alternatives in the description 😎